app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -16,6 +17,9 @@ class Admin::AdminSetsController < ApplicationController
     self.list_search_builder_class = CurationConcerns::AdminSetSearchBuilder
 
     def show
+      # Even though the user can view this admin set, they may not be able to view
+      # it on the admin page.
```

Comment:
Could the same be true for the index action too (and, if so, should this get pulled up to a `before_action` callback?)